DataFormats/SoATemplate/interface/SoACommon.h

```diff
@@ -555,7 +565,8 @@ namespace cms::soa {
 
   // Helper function to compute aligned size
   constexpr inline byte_size_type alignSize(byte_size_type size, byte_size_type alignment) {
-    return ((size + alignment - 1) / alignment) * alignment;
+    return ((size + alignment - 1) / alignment) *
+           alignment;  //this is an integer division -> it rounds size to the next multiple of alignment
```

Comment:
Can you move the comment to the line above the `return`, to avoid the line wrap ?